discord_bot/bot.py

```diff
@@ -37,8 +37,11 @@ async def my_long_command_function(ctx: interactions.SlashContext):
   opt_type=interactions.OptionType.STRING,
   autocomplete=True,
 )
-async def ask_model(ctx: interactions.SlashContext, model: str):
-  await ctx.send(f"You asked model {model}")
+@interactions.slash_option(
+  name="prompt", description="Enter your prompt", required=True, opt_type=interactions.OptionType.STRING
```

Comment:
Maybe specify a sensible min_length to avoid wasting an API call on accidentally send off an incomplete prompt

Reply:
make it >=10 chars?